tests/performance/specs/home-block-theme.test.js

```diff
@@ -21,7 +21,9 @@ describe( 'Server Timing – Twenty Twenty Three', () => {
 	} );
 
 	afterAll( async () => {
-		const resultsFilename = basename( __filename, '.js' ) + '.results.json';
+		const prefixArg = process.argv.find((arg) => arg.startsWith('--prefix'));
+		const fileNamePrefix = prefixArg ? `${prefixArg.split('=')[1]}-` : '';
+		const resultsFilename = fileNamePrefix + basename( __filename, '.js' ) + '.results.json';
```

Comment:
Can we make this rather complicated logic a reusable utility function, e.g. like `getResultsFilename( specFile )`? It's in the two specs, and as we add more specs in the future it'll see more and more usage.

Reply:
Thanks Felix for the review. I have added new function https://github.com/10up/wordpress-develop/pull/36/commits/167250397ea8d8be8508fb21773244aad48af36e